MDC2020/POMS/catAndShuf.sh

```diff
@@ -58,8 +58,9 @@ do
         numbers=(${sequencer//_/ })
         counter_string_old=${numbers[1]}
         counter_string=$(printf "%08d" $ifile)
-        sed -i "s/$counter_string_old/$counter_string/g" $f
-        sed -i "s/$counter_string_old/$counter_string/g" $f.json
+        sed -i "/outputs/s/$counter_string_old/$counter_string/g" $f
+        sed -i "/dh.sequencer/s/$counter_string_old/$counter_string/g" $f.json
+        sed -i "/file_name/s/$counter_string_old/$counter_string/g" $f.json
         mv "$f" `basename "${f/$counter_string_old/$counter_string}"`
```

Comment:
Can we test this script using MDC2020i output?

Reply:
The script runs fine interactively (it was running fine also before) but I caught a bug where it was replacing also one of the input filenames (and it shouldn't). The only way to really test this is to run on the grid because only there we can verify the SAM declaration.